wormhole/proxy/wormhole.go

```diff
@@ -184,16 +184,18 @@ func (ws *WormholeService) getHttpsProxyUrl(ctx context.Context, agentGroup *Age
 		port = ws.SessionOverrides.Port
 	}
 
-	httpsProxyUrl := fmt.Sprintf("socks5://%s:%s@%s:%d", user, password, host, port)
-	return httpsProxyUrl, nil
+	return fmt.Sprintf("socks5://%s:%s@%s:%d", user, password, host, port), nil
 }
 
 func (ws *WormholeService) getProxyConfig(ctx context.Context, agentGroup *AgentGroup) (*httpproxy.Config, error) {
-	httpsProxyUrl, err := ws.getHttpsProxyUrl(ctx, agentGroup)
+	proxyURL, err := ws.getProxyURL(ctx, agentGroup)
 	if err != nil {
 		return nil, err
 	}
-	return &httpproxy.Config{HTTPSProxy: httpsProxyUrl}, nil
+	return &httpproxy.Config{
+		HTTPProxy:  proxyURL,
```

Comment:
This means that if the transport is being used for an HTTP request (not just HTTPS), we should still use the proxy.

Reply:
By default we're going to assume that the customer's service is serving HTTP, not HTTPS. Our preview URL will be HTTPS but we're terminating SSL on our side.